### PR TITLE
[IMP] web_pivot_computed_measure: Time Ranges Comparison

### PR DESCRIPTION
**Pull request by Tardo**
Adds support to use "Time Ranges":
time_Ranges

cc @tecnativa TT23282

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: web_pivot_computed_measure/static/src/js/pivot_model.js [hunk 1]
```diff
@@ -102,29 +103,78 @@ odoo.define('web_pivot_computed_measure.PivotModel', function (require) {
...
         },
 
+        /**
+         * Helper function to add computed measure fields data into 'dataPoint'
+         *
+         * @param {Object} dataPoint
+         * @param {Object} dataPointComp
+         */
+        _fillComputedMeasuresData: function (dataPoint, dataPointComp) {
+            for (var cm of this._computed_measures) {
```
Comment (yajo) [inline web_pivot_computed_measure/static/src/js/pivot_model.js]: This syntax is not available in the ES version supported in Odoo 12: https://travis-ci.org/github/OCA/web/jobs/670341645#L1434

**Review by yajo — CHANGES_REQUESTED** [review]
Just a little detail. Good overall.

**Comment by Tardo** [comment]
Changes done!

**Review by yajo — APPROVED** [review]

**Pull request merged**